.codeclimate.yml

```diff
@@ -0,0 +1,9 @@
+languages:
+  Ruby: false
+  JavaScript: true
```

Comment:
Although we're converting them to JS, we also have quite a few coffeescript files.